chromium_src/check_chromium_src_config.json5

```diff
@@ -116,6 +116,9 @@
             "TargetGetModuleFileNameExA64",
             "TargetGetModuleFileNameExW64",
         ],
+        "ui/color/color_id.h": [
+            "MATERIAL_MISSING_COLOR_IDS"
+        ],
```

Comment:
Q. Is this for ignoring warning saying something like "no symbol was found in the associated files?"

Reply:
Nah, its for not triggering PRESUBMIT because I don't undefine the macro again at the end of the file

Reply:
Aha! Thanks